spec-dtslint/operators/merge-spec.ts

```diff
@@ -0,0 +1,147 @@
+import { of, asyncScheduler } from 'rxjs';
+import { merge } from 'rxjs/operators';
+
+it('should accept no parameter', () => {
+  const a = of(1, 2, 3).pipe(merge()); // $ExpectType Observable<number>
+});
+
+it('should infer correctly with scheduler param', () => {
+  const a = of(1, 2, 3).pipe(merge(asyncScheduler)); // $ExpectType Observable<number>
+});
+
+it('should infer correctly with concurrent param', () => {
+  const a = of(1, 2, 3).pipe(merge(3)); // $ExpectType Observable<number>
+});
+
+it('should infer correctly with concurrent and scheduler param', () => {
+  const a = of(1, 2, 3).pipe(merge(3, asyncScheduler)); // $ExpectType Observable<number>
+});
+
+it('should infer correctly with 1 Observable param', () => {
+  const a = of(1, 2, 3);
+  const b = of('a', 'b', 'c');
+  const res = a.pipe(merge(b)); // $ExoectType Observable<number|string>
```

Comment:
Typos throughout this PR: `ExpoectType` => `ExpectType` 

:)

Reply:
Opps... sorry @benlesh 
Fixed now.